### PR TITLE
chore&fix(plugins): replace undotree with telescope-undo.

### PR DESCRIPTION
**Pull request by ayamir**

### CODE REVIEW

File: lua/keymap/init.lua [hunk 1]
```diff
@@ -73,9 +73,8 @@ local plug_map = {
 	["n|<C-n>"] = map_cr("lua toggle_nvimtree()"):with_noremap():with_silent(),
 	["n|<Leader>nf"] = map_cr("NvimTreeFindFile"):with_noremap():with_silent(),
 	["n|<Leader>nr"] = map_cr("NvimTreeRefresh"):with_noremap():with_silent(),
-	-- Plugin Undotree
-	["n|<Leader>u"] = map_cr("UndotreeToggle"):with_noremap():with_silent(),
 	-- Plugin Telescope
+	["n|<Leader>u"] = map_cr("lua require('telescope').extensions.undo.undo()"):with_noremap():with_silent(),
```
Comment (CharlesChiuGit) [inline lua/keymap/init.lua:77]: Maybe "\<leader\>fu" to fit fuzzy find naming conventions?
And add it to legendary?

Also, is there a reason not using "Telescope undo"? Is this way faster?

**Review by CharlesChiuGit — COMMENTED** [review]
Some minor questions

File: lua/keymap/init.lua (hunk 1 above)
Comment (ayamir) [reply lua/keymap/init.lua:77]: 1. `<leader>u` to keep coherence with original keymap with undotree.
2. I will add others keymaps to legendary for next commit after merging.
3. Just keep consistent with other external extensions.

**Review by CharlesChiuGit — APPROVED** [review]

File: lua/keymap/init.lua (hunk 1 above)
Comment (CharlesChiuGit) [reply lua/keymap/init.lua:77]: cool! got it!

**Pull request merged**